frontend/src/app/modules/posts/components/post/post/post.component.ts

```diff
@@ -15,6 +15,10 @@ export class PostComponent implements OnChanges, OnInit{
   userId: string = '';
   isEdited: boolean = false;
 
+  @Output() contentUpdated: EventEmitter<PostGet> = new EventEmitter<PostGet>();
+
+  @Output() deleted: EventEmitter<string> = new EventEmitter<string>();
```

Comment:
wiem że to takie przypierdalanie się, ale najfajniej byłoby jakiś porządek mieć w tych komponentach. Input, Output itp. na górze, potem constructor z fieldami i na dole funkcje

Reply:
patrz zróbmy może tak i się tego od teraz trzymajmy:
* Input, Output itp.
* properties
* constructor
* funkcje prywante
* funkcje angularowe typu OnInit, OnDestroy itp.

może być taka kolejność?

Reply:
dobra mozemy tego sie trzymac